homeassistant/components/logbook.py

```diff
@@ -469,6 +469,14 @@ def _exclude_events(events, entities_filter):
             domain = event.data.get(ATTR_DOMAIN)
             entity_id = event.data.get(ATTR_ENTITY_ID)
 
+        elif event.event_type == EVENT_AUTOMATION_TRIGGERED:
+            domain = event.data.get(ATTR_DOMAIN)
```

Comment:
The new events don't have a domain attribute on the event data.  This can be hardcoded to `automation`.